To_do_list/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for To_do_list project.
+
+Generated by 'django-admin startproject' using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from email.mime import base
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-^mh1rgihwn4x7tl@6)%2io-3-%4jk9!8fd^h-jp*sxr#d9a8!_'
```

Comment:
Secrets should not be kept in the repository. Rather we should use the environment file for it